src/orion/benchmark/benchmark_client.py

```diff
@@ -65,9 +73,24 @@ def get_or_create_benchmark(
 
     benchmark = _create_benchmark(name, algorithms, targets)
 
+    if input_configure and input_benchmark != benchmark.configuration:
+        logger.warn(
+            "Benchmark with same name is found but has different configuration, "
+            "which will be used for this creation.\n{}".format(benchmark.configuration)
+        )
+
     if benchmark_id is None:
-        # persist benchmark into db
-        _register_benchmark(benchmark)
+        logger.debug("Benchmark not found in DB. Now attempting registration in DB.")
+        try:
+            _register_benchmark(benchmark)
+            logger.debug("Benchmark successfully registered in DB.")
+        except DuplicateKeyError:
+            print("exception....")
```

Comment:
This `exception` print may be misleading and scary for users. I think we can remove it. The logging below could be `info` instead of `debug`.